test/utility/parse_opts.c

```diff
@@ -57,6 +57,103 @@ parse_seed(uint64_t *seedp, const char *seed_str, char **parse_end)
     *seedp = (uint64_t)strtoll(seed_str, parse_end, 10);
 }
 
+/*
+ * parse_delay_error --
+ *     Parse artificial delays and errors seed from the command line.
+ */
+static void
+parse_delay_error(uint32_t *seedp, const char *seed_str, char **parse_end)
+{
+    *seedp = (uint32_t)strtol(seed_str, parse_end, 10);
+}
+
+/*
+ * parse_tiered_comma_separated_options --
+ *     Parse a command line option for the tiered storage configurations.
+ */
+static void
+parse_tiered_comma_separated_options(const char *config_str, uint32_t *opts_key1,
+  uint32_t *opts_key2, const char *usage, bool insert_delays)
+{
+    char *parse_end;
+    const char *config_end;
+    bool delay_config_option;
+
+    delay_config_option = true;
+    /* Point the end of string either to comma or the final null character. */
+    config_end = strchr(config_str, ',');
+
+    if (config_end == NULL) {
+        config_end = &config_str[strlen(config_str)];
+        delay_config_option = false;
+    }
+
+    parse_end = (char *)config_end;
+    if (*opts_key1 != 0)
+        testutil_die(
+          EINVAL, "%s x,x option specified twice: %s", insert_delays ? "-Pd" : "-Pe", usage);
+    parse_delay_error(opts_key1, config_str, &parse_end);
+
+    if (delay_config_option == false) {
```

Comment:
More idiomatic to say `if (!delay_config_option)`

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5